packages/react/src/components/Table/Table.tsx

```diff
@@ -2,20 +2,27 @@ import React, { TableHTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-const Table = ({
-  children,
-  className,
-  ...other
-}: TableHTMLAttributes<HTMLTableElement>) => (
-  <table className={classNames('Table', className)} {...other}>
+interface TableProps extends TableHTMLAttributes<HTMLTableElement> {
+  variant: 'border' | undefined;
```

Comment:
Alternatively, we can make this optional:

```suggestion
  variant?: 'border';
```

Reply:
Indeed, this is better.